### DISCUSSION TITLE
Mixing C# markup and Xaml

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#17121] [Q&A] [answered]
Say I already have an existing app where the view is written in xaml. Would there be some potential problems if I wanted to add new pages in C# markup, or should it just work?

**Comment by dansiegel** [comment]
Can't say I've tested this exactly... but in theory it should work.

**Reply by jeromelaban** [reply, marked as answer]
@Marc-Antoine-Soucy The XAML tooling is always available, particularly because `App.xaml` is available. You should be able to use it without issues.
